api/arc.go

```diff
@@ -1416,26 +1476,61 @@ func ParsePOSTTransactionResponse(rsp *http.Response) (*POSTTransactionResponse,
 		}
 		response.JSON400 = &dest
 
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 409:
```

Comment:
I guess it's better to use constant `http.StatusConflict` instead of "magic number" here. What do you think ?

Ref: https://go.dev/src/net/http/status.go (seek StatusConflict)

Reply:
This code I did not edit. arc.go is generated from arc.yml.